devDungeon/src/systems/FallingSystem.java

```diff
@@ -82,12 +82,10 @@ private void teleportPlayerIfPossible() {
   }
 
   private Tile getSafeTile(Point heroCoords) {
-    Tile tile;
-    try {
-      tile = LevelUtils.accessibleTilesInRange(heroCoords, 5).getFirst();
-    } catch (NoSuchElementException e) {
-      tile = Game.randomTile(LevelElement.FLOOR);
-    }
-    return tile;
+    return LevelUtils.accessibleTilesInRange(heroCoords, 5).stream()
+        .findFirst()
```

Comment:
> Das List#getFirst passte schon - es holt das erste Element aus der Liste. Dein Code macht das auch, aber deutlich komplizierter :)

Funny, meine IDE hat laut gerufen "DAT GIBTS NICHT" und weigerte sich auch zu Compilieren....

Reply:
schräg. über gradle ist ja java21 sowohl als jdk als auch als sprach-/feature-version eingestellt. ist das in deinen projekteinstellungen irgendwie überschrieben? oder reichen die gradle-einstellungen noch nicht ganz aus?

Reply:
> Wieso frage ich hier eigentlich LevelUtils und nicht Game? (API)

Hab eine alternativen Call über Game eingeführt 

> schräg. über gradle ist ja java21 sowohl als jdk als auch als sprach-/feature-version eingestellt. ist das in deinen projekteinstellungen irgendwie überschrieben? oder reichen die gradle-einstellungen noch nicht ganz aus?

ich versteh es auch nicht ganz. Andere Java 21 Features frisst er ohne Probleme. Nur im Dev-Dungeon (konkret das `getFirst`) mag er nicht. Ich bin noch unentschlossen ob ich jetzt lusten hab meine IDE zu verstehen....

Reply:
GitHub mag getFirst auch nicht.... 

Anyway grade auch gelesen

> If this List is not empty, the implementation in this interface returns the result of calling get(0). Otherwise, it throws NoSuchElementException.

da muss ich den Aufruf etwas ändern. Das `getFirst()orElse` Konstrukt dürfte nicht funktionieren

Reply:
Guter Punkt. Wenn die `LevelUtils#accessibleTilesInRange` eine leere Liste liefert, knallt Dir `List#getFirst` weg.

Andererseits wäre das auch in dem Weg über `.stream().findFirst().orElse()` so, nur komplizierter ausgedrückt.

Wenn Du mich fragst, ich würde bei `getFirst` bleiben. In `FallingSystem#teleportPlayerIfPossible` gehört ein try/catch rein - der Name sagt ja: teleportiere, wenn möglich. Wenn es nicht möglich ist (aka `accessibleTilesInRange` eine leere Liste produziert), dann passiert eben nix.